test/functional/api/cas/cli.py

```diff
@@ -126,10 +128,12 @@ def stop_cmd(cache_id: str, no_data_flush: bool = False, shortcut: bool = False)
     return casadm_bin + command
 
 
-def list_cmd(output_format: str = None, shortcut: bool = False):
+def list_cmd(output_format: str = None, full_path: bool = True, shortcut: bool = False):
     command = " -L" if shortcut else " --list-caches"
     if output_format == "table" or output_format == "csv":
         command += (" -o " if shortcut else " --output-format ") + output_format
+    if full_path:
+        command += (" -f " if shortcut else " --full-path ")
```

Comment:
shouldn't it be '-b'? I looked into casadm help:  
-b  --by-id-path                    Display by-id path to disks instead of short form /dev/sdx

Reply:
Yeah, it will be, I'm still fighting. Thanks for reminder.